projects/ngrx.io/content/navigation.json

```diff
@@ -8,6 +8,10 @@
       "url": "docs",
       "title": "Docs"
     },
+    {
+      "url": "guide/migration/v7",
```

Comment:
Let's make this a side nav section called `Migrations` with a `V7` menu item under it.

Reply:
Something like this? 
![image](https://user-images.githubusercontent.com/28659384/47519866-e0024180-d88e-11e8-875e-364c6def169f.png)